### DISCUSSION TITLE
Please add an inbult bookmark manager as Startpage

### DISCUSSION THREAD

**Discussion by tasinone** [Floorp-Projects/Floorp#963] [Feature Request]
Hello Dev, I used to use Vivaldi as my main browser before Floorp because managing Bookmark is so easy on Vivaldi and also it makes Vivaldi unique from all other Chromium-based browsers. 
However, after searching a lot I have found an Addon that works nearly as Vivaldi's startpage. It has also the functionality to create different folders for different types of bookmarks just like Vivaldi.
Addon link: https://github.com/twildh/quick-start

It will be very helpful if you implement this Addon to Floorp by default. Not only that, but it will also make Floorp unique from other Firefox-based browsers. Although this Addon lacks some features such as changing the background and some others. But I have already created a ticket on to that repository which you can check here: https://github.com/twildh/quick-start/issues/10

After they patch these features please consider adding this extension to Floorp. I am looking forward to your response.

**Comment by Foxxsayy** [comment]
Have you considered using [raindrop.io](https://raindrop.io/download)? It's an incredible bookmark maanger - far beyond anything I've found in a browser. It's free, uses tagging, has a great search, folders, cross-platform (PC, OSX, Linux, iOS, and Android), and has quite a few other neat features such as the option to share a collection of bookmarks with friends or publicly, integrations, batch operations, etc.

All of that's totally free, and for less than $2.50/month, you can also add full-text searches, permanent offline page saves, reminders, annotations, and a good bit more. 

I highly recommend it.

**Comment by tasinone** [comment]
Yeah, I do use it for more than a year.

**Comment by SystemError12** [comment]
Similar suggestion already here:

[Startpage](https://github.com/Floorp-Projects/Floorp/discussions/684#discussion-6079721)
